### ISSUE TITLE
Alerts tool not replicating alerts from R alerts 

### ISSUE THREAD

**Issue by turley85** [AtlasOfLivingAustralia/alerts#243]
I've loaded a few lists now and I'm failing replicate the alerts that we're getting from R. 

For example, [BioSecurity alert for NSW_HRCC_Weeds](https://lists.ala.org.au/speciesListItem/list/dr27309) was loaded and I ran the alert backdated to 1 May 2024, which showed no results. 

I'm also having this issue with [BioSecurity alert for NSW_DPI_CaneToads_list](https://lists.ala.org.au/speciesListItem/list/dr27207) and [BioSecurity alert for AUS_ACPPO_list](https://lists.ala.org.au/speciesListItem/list/dr27205) where I can't replicate the alerts being sent from R. 

<img width="735" alt="image" src="https://github.com/user-attachments/assets/b3cd3efa-5648-4b9b-a90e-363b6bc5c272">
<img width="1782" alt="image" src="https://github.com/user-attachments/assets/ae8a63ee-9fea-4fd6-9006-4876635bb72f"><img width="1665" alt="image" src="https://github.com/user-attachments/assets/3b21d63e-b3d9-4ad5-b661-d86f81c37c6c">

**Comment by turley85** [comment]
example occurrences being missed: https://biocache.ala.org.au/occurrences/1edf0dce-ea3a-4dae-ad71-1e04ff2e3b6e

and https://biocache.ala.org.au/occurrences/2138fda1-581e-4bc7-ac55-9fc27da69820

**Comment by turley85** [comment]
This is occurring for another list just added[BioSecurity alert for NSW_NPWS_CentralTablelands_Weeds](https://lists.ala.org.au/speciesListItem/list/dr27312) - 

<img width="1451" alt="image" src="https://github.com/user-attachments/assets/aa03dd60-213b-4e75-b40f-8fc620a96927">


R alert here: 
<img width="1451" alt="image" src="https://github.com/user-attachments/assets/f9a762eb-ab94-4463-a75c-0adc7d12b667">

**Comment by turley85** [comment]
Same issue with NSW_NPWS_GreaterSydney_Weeds, [dr27315](https://lists.ala.org.au/speciesListItem/list/dr27315) 

Eg. should match to this occurrence: https://biocache.ala.org.au/occurrences/dc6646b1-0f4d-4899-9627-0b4d666b1cb4

**Comment by kylie-m** [comment]
Hi Andrew, as mentioned I have a couple of edits to try for some of these, and will need to get back to you soon on the others. 

- For [BioSecurity alert for AUS_ACPPO_list](https://lists.ala.org.au/speciesListItem/list/dr27205) can you please try replacing `cl927` with `cl927:*`? I've updated Anoplolepis gracilipes and that species now seems to be picked up in the alert, all within Australia & coastal waters

- For NSW_NPWS_GreaterSydney_Weeds, I noticed the [spatialObject:9433213](https://spatial.ala.org.au/?pid=9433213) for Heteranthera zosterifolia starts north of Taree but the occurrence record is in the Sydney suburbs so wouldn't be picked up by the alert in this case. May need to check the spatial ID https://spatial.ala.org.au/?pid=9433213

**Comment by turley85** [comment]
Thanks Kylie - cl927:* appears to work!

And I'll double check the other shape files - excel has added +1 to the spatial object number as I copied it in GreatSydney, which cause the issue there. I'll double check all the other lists for this error too.

**Comment by turley85** [comment]
@kylie-m looks NSW_NPWS_GreaterSydney_Weeds is still not working. 
[Update - BioSecurity alert for NSW_NPWS_GreaterSydney_Weeds.pdf](https://github.com/user-attachments/files/16276012/Update.-.BioSecurity.alert.for.NSW_NPWS_GreaterSydney_Weeds.pdf)

I re-uploaded the shapefile https://spatial.ala.org.au/?pid=9434716 and corrected the fq field in the relevant list here: https://lists.ala.org.au/speciesListItem/list/dr27315, however, when testing, it is still showing 0 new records.  

I believe it should match this occurrence, as it is within the shapefile area:  https://biocache.ala.org.au/occurrences/dc6646b1-0f4d-4899-9627-0b4d666b1cb4

**Comment by kylie-m** [comment]
Hi @turley85, we've done some more investigating as a team, and believe the "shape" column in the lists is interfering with the "fq"  column. Renaming the column slightly (eg "shapefile") should address the issue so that it functions as a 'notes' style field rather than trying to match something in the system. I've tested this with a variation of your [NSW DPI Cane Toads](https://lists.ala.org.au/speciesListItem/list/dr27207) list, and it's functioning well in Alerts (see my [Cane Toads](https://lists.ala.org.au/speciesListItem/list/dr27340) list). 

I've also attached the example csv upload file on the [Biosecurity Alerts workflow](https://confluence.csiro.au/display/ALASD/ALA+Biosecurity+Alerts+Workflow) Confluence page
